nautobot/core/tests/test_models.py

```diff
@@ -12,3 +18,47 @@ def clean(self):
     def test_validated_save_calls_full_clean(self):
         with self.assertRaises(ValidationError):
             self.FakeBaseModel().validated_save()
+
+    def test__content_type(self):
+        """
+        Verify that the ContentType of the object is cached.
+        """
+        self.assertEqual(self.FakeBaseModel._get_content_type(), self.FakeBaseModel._get_content_type(skip_cache=True))
```

Comment:
What content-type does `FakeBaseModel` have, anyway? :) Should we test this with a real model instead?

Reply:
Adding
```py
print(self.FakeBaseModel.get_content_type())
self.assertEqual(self.FakeBaseModel._content_type.id, 0)
```

Results in:
```
Stdout:
core | fake base model

AssertionError: 13 != 0
```